packages/blade/src/utils/useColorScheme/useColorScheme.ts

```diff
@@ -1,6 +1,7 @@
 import { useState, useCallback } from 'react';
-import { ColorSchemeNames, ColorSchemeNamesInput, colorSchemeNamesInput } from '../../tokens/theme';
+import { colorSchemeNamesInput } from '../../tokens/theme';
```

Comment:
why not use the `/tokens/index` reexport?

Reply:
because of this https://github.com/razorpay/blade/pull/312#discussion_r657679212